packages/config/src/chains/dev.ts

```diff
@@ -49,7 +49,16 @@ const mangataLocal = createChain({
 });
 
 const moonbaseLocal = createChain({
-  assets: [{ asset: DevTokens.moonbaseLocal, isNative: true }],
+  assets: [
+    { asset: DevTokens.moonbaseLocal, isNative: true },
+    {
+      asset: DevTokens.tur,
+      contractAddress: "0xfFffffFf6448d0746f2a66342B67ef9CAf89478E",
+      id: "133300872918374599700079037156071917454",
+      isNative: false,
+      otherSymbol: "xcTUR",
```

Comment:
Let’s remove the definition of `otherSymbol`, and define a new asset xcTur in `DevTokens`.